contracts/core/BlockStore.sol

```diff
@@ -493,6 +467,50 @@ contract BlockStore is BlockStoreInterface {
 
     /* Internal Functions */
 
+    /**
+     * @notice Marks a block in the block store as justified. The source and
+     *         the target are required to know when a block is finalised.
+     *         Only the polling place may call this method.
```

Comment:
Now it's internal 🤔

Reply:
Yes, it's internal `justify_ ` function. We also have an external function `justify` that is called by `onlyPollingPlace` and it internally calls  `justify_ `

Reply:
My comment was in regards to the documentation that reads "Only the polling place may call this method."

Reply:
😁  removed now!

Reply:
I still see it 🤔